src/main/java/com/quizapp/enterprise/services/GameService.java

```diff
@@ -16,7 +16,10 @@
     public Game startNewGame(int quizId) {
 
         Game newGame = new Game();
-        newGame.setGameCode(UUID.randomUUID().toString().substring(0,5));
+        newGame.setGameCode(UUID
+                .randomUUID()
+                .toString()
+                .substring(0, 5));
```

Comment:
reformatted the streams to use indentation so the code may be cleaner and easier to read.
This recommended by the "Iterators and Streams" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3